test/function.localstack.test.ts

```diff
@@ -50,23 +51,25 @@ interface TestFunctionResource {
     payload?: I | ((context: Outputs) => I)
   ): void;
 
-  skip: <I, O, Outputs extends Record<string, string> = Record<string, string>>(
+  skip: <
+    I,
+    O, // Forces typescript to infer O from the Function and not from the expect argument.
```

Comment:
How/why?

Reply:
1. How - I am not sure, but it worked. 
2. Why - Because I was unable to represent some tests and cases when inferring from the expected result.